### PR TITLE
UWC : Modified the tags of the containers

### PR DESCRIPTION
**Pull request by aditisa1**
Earlier containers were referring to EII_VERSION tag modified it to
UWC_VERSION.

Signed-off-by: aditisa1 <aditi.saxena@intel.com>

### CODE REVIEW

File: modbus-master/modbus_RTU/docker-compose.yml [hunk 1]
```diff
@@ -29,12 +29,13 @@ services:
       context: $PWD/../uwc/modbus-master/
       dockerfile: $PWD/../uwc/modbus-master/Dockerfile_RTU
       args:
+        UWC_VERSION: ${UWC_VERSION}
         EII_VERSION: ${EII_VERSION}
```
Comment (vkb1) [inline modbus-master/modbus_RTU/docker-compose.yml:33]: You can remove EII_VERSION in Dockerfile and compose.yml files as you don't need them.

**Review by vkb1 — CHANGES_REQUESTED** [review]
Rest all looks good.

File: modbus-master/modbus_RTU/docker-compose.yml (hunk 1 above)
Comment (aditisa1) [reply modbus-master/modbus_RTU/docker-compose.yml:33]: Hi Vinod,
EII_VERSION is being used in Dockerfile for ia_common 
https://github.com/open-edge-insights/uwc/blob/cacbbd0e76a54a43d8e6cb6d14c7f124996a617c/modbus-master/Dockerfile_RTU

**Review by vkb1 — APPROVED** [review]

**Comment by nagdeepgk** [comment]
@aditisa1 https://github.com/open-edge-insights/uwc/blob/master/docker-compose_UT.yml UT files also needs to be updated from EII_VERSION to UWC_VERSION.

**Review by nagdeepgk — APPROVED** [review]

**Pull request merged**